code/game/objects/structures/bedsheet_bin.dm

```diff
@@ -305,6 +305,11 @@ LINEN BINS
 	extinguish()
 	update_icon()
 
+/obj/structure/bedsheetbin/wrench_act(mob/user, obj/item/I)
+	if(user.a_intent == INTENT_HARM)
```

Comment:
Just curious, why check for harm intent specifically and not just have it happen on any intent?

Reply:
In case you want to hide the wrench inside the linen bin.

Reply:
Ah, didn't see that.